docs/service-guide/create-service.rst

```diff
@@ -32,72 +32,16 @@ If you are using the FastAPI template, tagging in this fashion is required since
 Create the Docker image
 =======================
 
-The Docker image can be stored in any container registry that is usable by Kubernetes, but for Rubin-developed services, we normally use DockerHub.
-(We may switch to the Google Container Registry later, but for now DockerHub is used for all images.)
+The Docker image can be stored in any container registry that is usable by Kubernetes, but for Rubin-developed services using the FastAPI template, we usually push both to the `GitHub Container Registry <https://ghcr.io>`__ and Docker Hub.  Google Artifact Registry is in play for Science Platform images and may eventually be used more widely.  We may eventually stop publishing to Docker Hub; our workflow is centered on GitHub and the long-term future of Docker-the-company does not look very secure.
 If your image must be stored in a private container registery, the credentials for that registry must be added to the pull secret.
 
 If you use the FastAPI service template, a ``Dockerfile`` will be created as part of the new repository template, and GitHub Actions will be set up in the new repository to build and push new Docker images for tagged releases.
-To enable this workflow, you must create two secrets in your new GitHub repository, ``DOCKER_USERNAME`` and ``DOCKER_TOKEN``.
-``DOCKER_USERNAME`` should be set to the DockerHub username of the account that will be pushing the new Docker images.
+
+If you use ghcr.io as your repository (which is the FastAPI template default) you require a repository secret named ``GITHUB_TOKEN``.
+If you are using Docker Hub you must create two secrets in your new GitHub repository, ``DOCKER_USERNAME`` and ``DOCKER_TOKEN``.
+``DOCKER_USERNAME`` should be set to the Docker Hub username of the account that will be pushing the new Docker images.
 ``DOCKER_TOKEN`` should be set to a secret authentication token for that account.
 We recommend creating a separate token for each GitHub repository for which you want to enable automatic image publication, even if they all use the same username.
 
-You may need to have a Docker Pro or similar paid DockerHub account.
+If using Docker Hub You may need to have a Docker Pro or similar paid Docker Hub account.
 Alternately, you can contact SQuaRE to set up Docker image publication using our Docker account.
```

Comment:
Maybe close out this page with a one-sentence paragraph that then links to the `add-service.rst` page. Something along the lines of:

> Next, deploy your service by create a Helm chart in Phalanx.

Reply:
Done (also a pointer to the intermediate "add a secret" step).